step.sh

```diff
@@ -71,11 +71,16 @@ else
 	echo "The gradlew wrapper was not found. Please provide the correct project_location."
 	exit 1
 fi
-if [ "${build_variant}" = "debug" ]
+
+basename="uploadToAppSweep"
+if [ -z "${build_variant}" ];
 then
-	output=$($GRADLEW uploadToAppSweepDebug)
+  # Build variant absent, we default to release
+  output=$($GRADLEW "${basename}Release")
 else
-    output=$($GRADLEW uploadToAppSweepRelease)
+  # Capitalize first letter of variant name, concatenate with Gradle task basename
+  capitalized="$(echo ${build_variant} | awk '{print toupper(substr($0, 1, 1)) substr($0, 2)}')"
```

Comment:
Is that solely so that it looks nicer in the AS frontend?

Reply:
Yesn't - its main purpose is to form the Gradle task name correctly, so for example, for a `debug` build variant, the Gradle Task name auto-generated by the AppSweep plugin is `uploadToAppSweepDebug`. Making it look better in the UI is just a nice side-effect.